app/models/user.rb

```diff
@@ -3,7 +3,10 @@ class User < ApplicationRecord
   has_many :comments
   has_many :likes
```

Comment:
According to the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md)

- [ ] We should add unit tests for all of our models, `User`, `Post`, `Comment`, and `Like`
- [ ] Please add unit specs for the `Comment`, and `Like` models 💪👍

Reply:
Then we can work on those issues first before progressing any further. The main advantage of this is when the issues are resolved at an early stage, no other part of the software is impacted. 

This leads to increased efficiency, reduced downtime, and reduced costs that would otherwise arise as a result of the whole design process stalling.

Reply:
- Hey @mohammadnewton, as requested by the previous reviewer, you should add unit test for ALL YOUR MODELS  to meet the project requirement, however, you've only added just two models, (user and post).

   - I can see you insist the test is meant for two models, well, I'd like you to consider answering these few questions I'd highlight below.
   1) how many models do you have in your database? 
   11) if there are four, what happens to the remaining two models you left with no test added to them?
   
  *This instruction asks you to provide a test for the four models, Please, make the requested changes* 
![image](https://github.com/mohammadnewton/blog_app/assets/59397159/04948c1d-d8c2-4925-b06f-2452dfef6aea)